### PR TITLE
COP 1976: Refactor MRZ Table

### PR DESCRIPTION
**Pull request by JimmySunDance**
## Description
Refactoring of MRZ table component

## To test
- Install app and run dev (this will open an electron app window)
- If this page is blank, refresh the page with cmd + r (mac) or ctl + r (windows)
- The MRZ data table will display on the "Data read from chip and MRZ" tab.

## Developer Checklist
\* Required

- [x] Up-to-date with master branch? \*

- [ ] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

### CODE REVIEW

File: packages/react/src/Components/Atoms/MRZTableHeading.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Global imports
```
Comment (Ali-Baker) [inline packages/react/src/Components/Atoms/MRZTableHeading.jsx:1]: This can't be atom as it's specific to MRZ

Comment (JimmySunDance) [reply packages/react/src/Components/Atoms/MRZTableHeading.jsx]: Moved to molecules.

File: packages/react/src/Components/Molecules/index.js [hunk 2]
```diff
@@ -6,5 +6,8 @@ export { default as LiveImage } from './LiveImage';
 export { default as MainSection } from './MainSection';
 export { default as MatchValue } from './MatchValue';
 export { default as MRZPanel } from './MRZPanel';
+export { default as MRZTableHeading } from './MRZTableHeading';
```
Comment (Ali-Baker) [inline packages/react/src/Components/Molecules/index.js:9]: Why don't you make this just `Heading` molecule?

We should be able to reuse Atoms & Molecules. MRZTableHeading, MRZTableRow and MRZTable are too specific to MRZ.

File: packages/react/src/Components/Molecules/MRZTableRow.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const MRZTableRow = ({ heading, chipData, MRZData }) => {
+  return (
+    <tr className="govuk-table__row">
+      <th className="govuk-table__cell" scope="row">
+        {heading}
+      </th>
+      <td className="govuk-table__cell">{chipData}</td>
```
Comment (Ali-Baker) [inline packages/react/src/Components/Molecules/MRZTableRow.jsx:11]: Atomise this

File: packages/react/src/Components/Molecules/MRZTableRow.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+// Global imports
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const MRZTableRow = ({ heading, chipData, MRZData }) => {
+  return (
+    <tr className="govuk-table__row">
```
Comment (Ali-Baker) [inline packages/react/src/Components/Molecules/MRZTableRow.jsx:7]: Atomise this

File: packages/react/src/Components/Molecules/MRZTableRow.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// Global imports
```
Comment (Ali-Baker) [inline packages/react/src/Components/Molecules/MRZTableRow.jsx:1]: Make this `TableRow.jsx` molecule

File: packages/react/src/Components/Molecules/MRZTable.jsx [hunk 6]
```diff
@@ -1,54 +1,39 @@
 // Global imports
```
Comment (Ali-Baker) [inline packages/react/src/Components/Molecules/MRZTable.jsx:1]: I think this can be Organ? What do you think?

**Pull request closed without merging**